src/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -141,6 +141,9 @@
     <Compile Include="$(SourceDir)\System\Xml\XmlReservedNs.cs">
       <Link>System\Xml\XmlReservedNs.cs</Link>
     </Compile>
+    <Content Include="dotnet-Microsoft.XmlSerializer.Generator.runtimeconfig.json">
```

Comment:
It'd be better to put the element in a separate `<ItemGroup>` as this one is for `<Compile>` elements.